02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const router = express.Router();
+
+// const { people } = require("../data.js");
+
+const { 
+      addPerson, 
+      getPeople,
+      getPersonById,
+      updatePerson,
+      deletePerson
+     } = require("../controllers/people.js");
+
+
+/*
+You then need to add a router.get() statement for the "/" path. 
+This should do the same thing as your app.get("/api/v1/people") statement. 
+Similarly, you need a router.post() statement for "/". Finally, at the bottom,
+ you need module.exports = router. 
+*/
+
+// router.get('/', (req, res) => {
+//   res.status(200).json({ success: true, data: people });
+// });
+
+// router.post("/", (req, res) => {
```

Comment:
Your `router.put` endpoint can use similar logic as the POST one, since both are updating People - the PUT will just edit an existing entry